src/panels/lovelace/common/directives/long-press-directive.ts

```diff
@@ -19,7 +16,7 @@ interface LongPressElement extends Element {
 
 class LongPress extends HTMLElement implements LongPress {
   public holdTime: number;
-  protected ripple: any;
+  public ripple: any;
```

Comment:
why public?

Reply:
After updating the deps it complained that the base was public and our was protected...

https://github.com/material-components/material-components-web-components/commit/e19f6d1df01ee071baa8cf03a03787d17bc0851a